src/app/containers/MediaPlayer/index.jsx

```diff
@@ -146,7 +130,11 @@ const MediaPlayerContainer = ({
     path(['media', 'contentExpired'], translations) ||
     contentNotAvailableMessage;
 
-  if (!available) {
+  const mediaIsValid = available && (versionId || blockId);
+  if (!mediaIsValid) {
+    if (!(versionId || blockId)) {
```

Comment:
Is it worth simplifying this to be `if (!available)`? 
+ easier to read / understand
- negated logic in if statement

Reply:
Not sure which part you mean! We need to check that it is both available and has one of versionId or blockId. If it's missing one of versionId/blockId, we log that it's missing. If it has one of blockId/versionId but is not available, we don't want to log out the missing value.

Reply:
Sorry, this is not very clear. 

Instead of `if (!(versionId || blockId)) {` could it be `if (!available) {`

Reply:
If it's not available, we don't want to use `logMissingMediaId`. We only want to use `logMissingMediaId` if there isn't an ID

Reply:
Oops I meant the other way around(available instead of !available). Anyways, let's just leave it. Have approved. :shipit:

Reply:
Oops I meant the other way around(available instead of !available). Anyways, let's just leave it. Have approved. :shipit: